orders/views.py

```diff
@@ -1,3 +1,109 @@
-from django.shortcuts import render
+import json
+import uuid
 
-# Create your views here.
+from django.http    import JsonResponse
+from django.views   import View
+
+from orders.models  import Cart, Order, OrderItem, OrderStatus, ItemStatus
+from accounts.utils import user_validator
+
+class OrderView(View):
+    @user_validator
+    def post(self, request):
+        try:
+            carts = Cart.objects.filter(account=request.user)
+
+            if carts:
```

Comment:
```suggestion
            if carts.exists():
```
존재 유무 같은 경우 exists()을 사용해 주세요.

Reply:
수정하였습니다.